components/script/dom/gpudevice.rs

```diff
@@ -190,6 +191,27 @@ impl GPUDevice {
         let _ = self.eventtarget.DispatchEvent(ev.event());
     }
 
+    /// <https://gpuweb.github.io/gpuweb/#abstract-opdef-validate-texture-format-required-features>
+    ///
+    /// Also returns wgpu's `TextureFormat` as it already does conversion inside
```

Comment:
I think this would be clearer: "Validates that the device suppports required features, and if so returns an ok containing wgpu's `TextureFormat`"